Makefile.3ds

```diff
@@ -38,7 +38,7 @@ APP_TITLE       := Super ftpd II Turbo
 APP_DESCRIPTION := v$(VERSION)
 APP_AUTHOR      := mtheall
 
-ICON		:=	meta/icon.png
+APP_ICON	:=	meta/icon.png
```

Comment:
I'm sorry, I intended to only have the change in the Makefile not the Makefile.3ds. I was not aware it would also grab the Makefile.3ds which I committed after making the pull request. Sorry!